jenkins-jobs/mci/project.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+#
+# Copyright (C) 2017 Bhushan Shah <bshah@kde.org>
```

Comment:
It occurs to me this may have other people's copyright as well :P

Reply:
I was not entirely sure, how to handle "derived" code licensing.. That's why I added line saying this file based on jenkins-jobs/nci/project.rb

Reply:
Always copy original copyright holders. That's generally applicable to any project really, references to files break over time making copyright holders substantially less accessible than they should be arguably breaking the "acknowledge original author" point in many free software licenses.

Reply:
Ack fixing it for this and also, job updater for MCI then.